vhdbuilder/packer/components.json

```diff
@@ -127,7 +127,8 @@
       "multiArchVersions": [
         "ciprod05192022",
```

Comment:
We can remove old images ciprod05192022 &  ciprod06272022 images.

Reply:
done